modules/oe_theme_helper/src/MetadataEvents.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+declare(strict_types = 1);
+
+namespace Drupal\oe_theme_helper;
+
+/**
+ * Contains all events related to the metadata plugin.
+ */
+final class MetadataEvents {
+
+  /**
+   * Name of the event fired during collecting info for metadata building.
+   *
+   * @Event
+   *
+   * @var string
+   */
+  const NODE = 'metadata_events.node';
```

Comment:
Let's call this actually with a bit more namespace: `page_header_metadata_events.node`